example.py

```diff
@@ -17,15 +19,19 @@
         'test_images',
         target_size=(256, 256),
         batch_size=1,
+        shuffle=False,
         class_mode='binary',
         subset='training')
 
 # 3 - Predict
-X, y = generator.next()
-print('Predicted :', classifier.predict(X), '\nReal class :', y)
+num_iterations = 0
```

Comment:
For this change in the predict file, I don't really want to provide a loop, this was a minimal working code to show that this uses the tf/keras framework, but there are many ways to loop over the images. So I would leave this file unchanged while letting the new script `predict_on_directory` do want you want to do.